cypress/integration/v2/blocks/icon.spec.js

```diff
@@ -382,4 +379,219 @@ describe( 'Icon Block', () => {
 			},
 		} )
 	} )
+
+	it( 'should adjust tablet options inside style tab', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'ugb/icon' )
+		cy.openInspector( 'ugb/icon', 'Style' )
+		cy.changePreviewMode( 'Tablet' )
+
+		// Test General options
+		cy.collapse( 'General' )
+		cy.adjust( 'Number of Icons / Columns', 6 )
+		cy.get( 'div.ugb-icon__content-wrapper' ).find( '.ugb-icon__item6' )
+
+		const aligns = [ 'left', 'center', 'right' ]
+		aligns.forEach( align => {
+			cy.adjust( 'Align', align, { viewport: 'Tablet' } ).assertComputedStyle( {
+				'.ugb-inner-block': {
+					[ `text-align` ]: align,
+				},
+			} )
+		} )
+
+		cy.changePreviewMode( 'Tablet' )
+
+		// Test Icon options
+		cy.collapse( 'Icon' )
+		cy.contains( 'Shaped' ).click( { force: true } )
```

Comment:
we might need to do something inside our `adjust` command instead of using `contains` here